tenderly-suite/handle-failed-tx-action/handle-failed-tx/handleFailedTx.js

```diff
@@ -0,0 +1,111 @@
+const axios = require('axios').default;
+const { ethers } = require('ethers');
+
+const WARNING_THRESHOLD = 10; // TODO: discuss for production
+const CRITICAL_THRESHOLD = 20;
+const TIME_SPLIT = 5 * 3600 * 1000; //  5 hours in milliseconds
+const PAGER_DUTY_ALERT_URL = 'https://events.pagerduty.com/v2/enqueue';
+
+const handleFailedTxFn = async (context, event) => {
+    const chainName = context.metadata.getNetwork();
+    const provider = new ethers.providers.JsonRpcProvider(await context.secrets.get(`RPC_${chainName.toUpperCase()}`));
+    const tx = await provider.getTransaction(event.hash);
+
+    const response = await provider.call(tx, tx.blockNumber);
+    const errorHash = response.slice(0, 10);
+    console.log('errorHash: ', errorHash);
+    let warningOptions = [];
+
+    switch (errorHash) {
+        case '0xdf359a15':
+            warningOptions = ['FlowLimitExceeded', 'TokenManager'];
+            break;
+        case '0xbb6c1639':
+            warningOptions = ['MissingRole', '-'];
```

Comment:
What is the significance of the '-' in this and the following two `warningOptions` arrays?